.github/workflows/integration-tests.yaml

```diff
@@ -38,13 +38,13 @@ env:
   JWT_SECRET: ${{ secrets.JWT_SECRET }}
   SEED_MAPS_API_KEY: ${{ secrets.SEED_MAPS_API_KEY }}
   TESTS_ENV: ${{ secrets.TESTS_ENV }}
-  LDAP_HOST: localhost
-  LDAP_PORT: 6379
-  LDAP_PASS: ${{ secrets.LDAP_PASS }}
-  LDAP_ORG: 'dc=covidsafepaths, dc=org'
-  LDAP_BIND: 'cn=admin, dc=covidsafepaths, dc=org'
-  LDAP_SEARCH: 'dc=covidsafepaths, dc=org'
-  LDAP_FILTER: '(&(objectClass=person)(cn={{username}}))'
+  AUTH_LOGOUT_REDIRECT_URL: 'http://127.0.0.1:5000'
+  AUTH0_BASE_URL: 'https://safeplaces.us.auth0.com'
+  AUTH0_CLIENT_ID: 'dXBl74NypszcDQQ2WPJsIUj99lSHcSMf'
+  AUTH0_CLIENT_SECRET: 'uagQlLvuLIJh_OFg5aR4lWRT3LhOyrp_qi1jHwgr4t0lLhOlktQAQo35SE3zOE1L'
+  AUTH0_API_AUDIENCE: 'https://safeplaces.cloud/'
+  AUTH0_CLAIM_NAMESPACE: 'https://safeplaces.cloud'
+  AUTH0_DB_CONNECTION: Test
```

Comment:
Same here.  Maybe use the same `${{ secrets.WHATEVER }}`

Reply:
Agreed, let's find a better way to handle these secrets